### ISSUE TITLE
Pin Tabs on Groups

### ISSUE THREAD

**Issue by eronmedeiros** [Drive4ik/simple-tab-groups#654]
I've tried to use the "Pin Tab" feature from Firefox, and it works when I have more then one group opened.
But if I close the group and reopens it, the pinned tabs just goes away and I lose those tabs.

It would be awesome to have Pin Tabs managed by the extension. Please consider this feature request.
I like the way Firefox dispose the Pinned Tabs, showing just the icon, not the text. So I think the new feature could be just like that.

**Comment by jtreagan** [comment]
I guess I have a slightly different problem.  My pinned tabs hang around, but are global over all my groups.  If I pin a tab in one group, it shows up in all my groups.  Likewise, if I delete a tab.  For me, pinned tabs are specific to the group I'm using.  I pin them to clean up my screen.  Otherwise, Simple-tab-groups does exactly what I want and is well-done.  However, this issue is important enough that I've started looking for an alternative.

**Comment by dkgv** [comment]
Did you find anything @jtreagan?

**Comment by grahamperrin** [comment]
Duplicate of #286

**Comment by jtreagan** [comment]
@dkgv Yes.  It a Firefox addon called "Tab Group Switch".  Seems to work pretty well.

**Comment by grahamperrin** [comment]
### [Tab Group Switch](https://addons.mozilla.org/addon/tab-group-switch/)

NB

>> … Form data, scrolling position, and tab history will be lost. …

**Comment by grahamperrin** [comment]
Why _abandon_ Simple Tab Groups for pinned tabs, when there are _complementary_ extensions that can manage the tabs?

**Comment by obvionaoe** [comment]
> Why _abandon_ Simple Tab Groups (to work with pinned tabs), when there are _complementary_ extensions that can manage pinned tabs?

which ones?
